amundsen_application/static/js/components/TableDetail/RequestMetadataForm/tests/index.spec.tsx

```diff
@@ -0,0 +1,106 @@
+import * as React from 'react';
+
+import { shallow } from 'enzyme';
+
+import { RequestMetadataForm, mapDispatchToProps, mapStateToProps, RequestMetadataProps } from '../';
+import globalState from 'fixtures/globalState';
+import { NotificationType } from 'interfaces/';
+
+const mockFormData = { 
+  'recipients': 'test1@test.com,test2@test.com',
+  'sender': 'test@test.com',
+  'table-description': 'on',
+  'fields-requested': 'off',
+  'comment': 'test',
+  get: jest.fn(),
+}
+mockFormData.get.mockImplementation((val) => {
+  return mockFormData[val];
+})
+// @ts-ignore: How to mock FormData without TypeScript error?
+global.FormData = () => (mockFormData);
+
+describe('RequestMetadataForm', () => {
+  const setup = () => {
+    const props: RequestMetadataProps = {
+      userEmail: '',
+      tableName: '',
+      tableOwners: [''],
+      submitNotification: jest.fn(),
+      requestIsOpen: false,
+      toggleRequest: jest.fn(),
+    };
+    const wrapper = shallow<RequestMetadataForm>(<RequestMetadataForm {...props} />)
+    return {props, wrapper}
+  };
+
+  describe('toggle', () => {
+    it('calls toggleRequest', () => {
+      const { props, wrapper } = setup();
+      const toggleRequestSpy = jest.spyOn(props, 'toggleRequest');
+      wrapper.instance().toggle();
+      expect(toggleRequestSpy).toHaveBeenCalled();
+    });
+  });
+
+  describe('submitNotification', () => {
+    it('calls submitNotification', () => {
+      const { props, wrapper } = setup();
+      const submitNotificationSpy = jest.spyOn(props, 'submitNotification');
+      wrapper.instance().submitNotification({ preventDefault: jest.fn() });
+      expect(submitNotificationSpy).toHaveBeenCalledWith(
+        mockFormData['recipients'].split(','),
+        mockFormData['sender'],
+        NotificationType.REQUESTED,
+        {
+          comment: mockFormData['comment'],
+          resource_name: props.tableName,
+          resource_url: window.location.href,
+          description_requested: mockFormData['table-description'] === "on" ? true : false,
```

Comment:
nit: your tests really shouldn't include boolean logic like this, a unit test is expected to be set up in such a way that given a particular input, we get a particular output. So we should just be expecting `true` or `false` here based on what condition we have set up.